ext/mbstring/mbstring.c

```diff
@@ -4226,13 +4226,6 @@ MBSTRING_API size_t php_mb_mbchar_bytes_ex(const char *s, const mbfl_encoding *e
 }
 /* }}} */
 
-/* {{{ MBSTRING_API size_t php_mb_mbchar_bytes() */
-MBSTRING_API size_t php_mb_mbchar_bytes(const char *s)
```

Comment:
Same here, we should rename php_mb_mbchar_bytes_ex.

Reply:
Understood, will do that.